library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -353,6 +362,41 @@ public Slice get(Slice key, long timeBucket, ReadSource readSource)
       }
     }
 
+
+    private int filteredCount = 0;
+    private int unfilteredCount = 0;
+    private long verifyBloomFilterBeginTime = 0;
+    private long verifyBloomFilterInterval = 5000;
```

Comment:
The reason I put them here is all of them in fact only used by method verifyBloomFilter()